### PR TITLE
csv-generator

### PR DESCRIPTION
**Pull request by slowmike**

### CODE REVIEW

File: database/csv-generator/csv-generator.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let fakeData = [];
+
+const getFakeData = () => {
```
Comment (tonykijak) [inline database/csv-generator/csv-generator.js:3]: Does this need a parameter? This function is called below with an argument.

File: database/csv-generator/faker-schema.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    name: faker.commerce.productName(),
+    detail: faker.lorem.slug(),
+    image: randImg,
+  };
+};
+
+const randImgList = (imgNum, totImg) => {
+  let result = new Set();
+  while(result.size < imgNum) {
+    result.add(Math.floor(Math.random()*totImg));
```
Comment (tonykijak) [inline database/csv-generator/faker-schema.js:16]: Can you use the shuffle deck algorithm to pick random images? It may be more efficient.

**Comment by tonykijak** [comment]
@slowmike I like how you're using a csv file to generate your db data.
In your images.csv, do you think the id/name fields are redundant? Are you planning on changing those?
Everything else looks good.

File: database/csv-generator/csv-generator.js (hunk 1 above)
Comment (slowmike) [reply database/csv-generator/csv-generator.js:3]: I was considering sending a value to decide how much data to give, but I decided it didn't matter and forgot to remove the argument from the function call.

File: database/csv-generator/faker-schema.js (hunk 2 above)
Comment (slowmike) [reply database/csv-generator/faker-schema.js:16]: How would i use shuffle deck to pick random images, not sure actually.

**Comment by slowmike** [comment]
@tonykijak I don't think the id/name fields are redundant, I just chose to use the id as the name for the sake of easier csv generation.

File: database/csv-generator/faker-schema.js (hunk 2 above)
Comment (tonykijak) [reply database/csv-generator/faker-schema.js:16]: I would refer to the solution to the toy problem. What it does is eliminate repeats, so you would not even need to use a set. However, your solution works, so it's low priority.

File: database/csv-generator/csv-generator.js (hunk 1 above)
Comment (tonykijak) [reply database/csv-generator/csv-generator.js:3]: got it. Looks good.

**Comment by tonykijak** [comment]
@slowmike If the ids are the names, then what is the purpose of an extra column for names? You can refer to the id column for its name. I might even go a step further to say that you know all the image names without the table (i.e. 1...100). Unless you are storing more information in the image table, I don't think you even need the table itself.

**Comment by slowmike** [comment]
I kept a separate column for the case of the image name not following the convention that I used in this case.

**Pull request merged**